### DISCUSSION TITLE
Comparisons of Execution Limits with nBomber

### DISCUSSION THREAD

**Discussion by PavanBodiPipeline** [VIP-Logic/NBomber#442] [Q&A]
We are trying to run a performance test with nBomber and hit the limit with execution. We are currently using free tier to run these tests. The test errors out with "maximum execution limit reached". Want to understand the limits with free tier and a comparison chart between free tiers and enterprise ones. Tried to find out documentation around the tier comparison and couldn't locate one.

Appreciate if there is any reference on this subject

**Comment by AntyaDev** [comment]
Hi @PavanBodiPipeline 

> The test errors out with "maximum execution limit reached". 

Can you please elaborate more?
I mean what LoadConfiguration do you have and what error message did you get?
NBomber doesn't have any predefined limits, it's up to the target machine that is running the NBomber test.

With NBomber Enterprise you will get an NBomber Cluster  + additional plugins.
